injectproxy/silences_test.go

```diff
@@ -74,11 +75,47 @@ func TestListSilences(t *testing.T) {
 			labelv:  []string{"default", "something"},
 			expCode: http.StatusUnprocessableEntity,
 		},
+		{
+			// Regex match
```

Comment:
hmm like with multiple label values, the silences API semantics don't allow to match multiple values.

https://github.com/prometheus-community/prom-label-proxy/blob/0664aa864a4e8aa5858d2772a0576cb3e385bb1f/injectproxy/routes.go#L305-L317

I'd recommend to update `assertSingleLabelValue()` and reject API calls when regex=true.

Reply:
`501 Not implemented` is now returned.